emulator/blockchain.go

```diff
@@ -700,27 +713,6 @@ func configureBootstrapProcedure(conf config, flowAccountKey flowgo.AccountPubli
 			fvm.WithStorageMBPerFLOW(conf.StorageMBPerFLOW),
 		)
 	}
-	if conf.TransactionFeesEnabled {
```

Comment:
this effectively removes this config parameter.

I'm ok with this, but lets note this on the PR description and also remove all references to it (including in the documentation), and maybe do a quick check in the community if anyone is using this.

Reply:
The config is still usable, in this piece of [code](https://github.com/onflow/flow-emulator/blob/master/emulator/blockchain.go#L544). Whether transaction fees are enabled or not, that shouldn't affect the computation and memory calculation. That's why I removed the `if conf.TransactionFeesEnabled`. Or did I misunderstand your point here?

Reply:
You are right! Sorry, missed that part.